File: beecrowd/bee1002/bee1002.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

int main()
{
    const double PI = 3.14159;

    double raio = 0;
    std::cin >> raio;

    double area = PI * pow(raio, 2);

    printf("A=%.4f\n", area);

    return 0;
}

File: beecrowd/bee1002/bee1002.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let raio = parseFloat(lines.shift());

const PI = 3.14159
let area = PI * Math.pow(raio, 2);

console.log(`A=${area.toFixed(4)}`);
